revo

cajubelt opened the pull request:
> To make a submission for the Bake Off, please complete the task lists
> below; otherwise remove them.

Your Dapp:

- [ ] connects to Valora via WalletConnect V1 or DAppKit;
- [ ] must be mobile-first or mobile responsive;
- [ ] is in one of the existing categories; and
- [ ] includes a link to user documentation.

Please confirm you have:

- [ ] read the [Dapp Bake Off Information](https://valoraapp.com/bakeoff); and
- [ ] read the [Dapp Bake Off Rules](https://valoraapp.com/bakeoff-rules).

aleksey-nov commented:
Hi team,
I have tried to zap in some Celo to mcUSD/mcEUR pool, but it doesn’t work for me. Nothing happens when I tap the button, no error message. I am ios, safari. Can you please fix it and let us know so we can review it again? Thanks!

jeanregisser commented:
@cajubelt Gentle reminder, you need to address the feedback we left above by Feb 28th if you wanna participate in the Bake Off.
Let me know if you have any question.

cajubelt commented:
Thanks @jeanregisser . We have added links to docs in 2 places (1 for new users, 1 for all users) and addressed the bug that I think Aleksey experienced.

We have a few more usability improvements we're hoping to get through this weekend, but if you want to re-review in the meantime you probly can @lisaw-valora @aleksey-nov . Keep in mind the liquidity is low right now since we are undergoing audits, but we plan to raise it this weekend as well.

jeanregisser commented:
Great I'll take a look soon!

**_Update_**: we've extended the deadline for bake Off participants to address our feedback. The new deadline is **March 3rd @ 11:59 pm PST** (UTC -8:00).
